notebook/notebook/notebook.py

```diff
@@ -27,7 +27,7 @@
 routes = web.RouteTableDef()
 
 # Must be int for Kubernetes V1 api timeout_seconds property
-KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 5))
+KUBERNETES_TIMEOUT_IN_SECONDS = float(os.environ.get('KUBERNETES_TIMEOUT_IN_SECONDS', 60))
```

Comment:
Don't change two things at once, please.  If we want the additional logging, let's put that in first and study what happened, make the timeout change, and compare.